tests/TestCodeGen.hs

```diff
@@ -41,18 +41,20 @@ codeGenTests = testGroup "Code generator unit tests"
   , camelCaseMessageFieldNames
   , don'tAlterEnumFieldNames
   , knownTypeMessages
+  , pythonInteroperation
+  ]
+
+pythonInteroperation :: TestTree
+pythonInteroperation = testGroup "Python interoperation" $ do
```

Comment:
I'm not sure I understand the connection between this test addition and the PR as a whole. Is there some connection or was it just something that happened to be roped in?

Reply:
Before we were failing to test all the different varieties of string representation.  I want to test all 3: `Data.Text.Lazy.Text`, `Data.Text.Text`, `Data.Text.Short.ShortText`.  Therefore I'm multiplying the number of tests by 3, and it is going to get verbose and error-prone to write out all those combinations by hand.  That's why I'm generating the list of tests now.

Reply:
Oh, I should mention that when large records are supported, we used to test only them, but now we also test regular records in that case.